ajax/libs/primer-markdown/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "primer-markdown",
+  "description": "GitHub stylesheets for rendering markdown.",
+  "filename": "build/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/primer/primer/tree/master/modules/primer-markdown"
+  },
+  "author": {
+    "name": "GitHub, Inc."
+  },
```

Comment:
Let's use one line here.
`"author": "GitHub, Inc."`
Because it's good to read.
Thanks.